### PR TITLE
Adding v2 tests and updating canvas identifier

### PR DESCRIPTION
**Pull request by glenrobson**

### CODE REVIEW

File: iiify/app.py [hunk 1]
```diff
@@ -139,6 +139,10 @@ def manifest3(identifier):
 
 @app.route('/iiif/<identifier>/manifest.json')
 def manifest(identifier):
+    return manifest2(identifier)
```
Comment (glenrobson) [inline iiify/app.py]: Default to v3.

File: iiify/app.py [hunk 2]
```diff
@@ -139,6 +139,10 @@ def manifest3(identifier):
...
+
+@app.route('/iiif/2/<identifier>/manifest.json')
+def manifest2(identifier):
     domain = purify_domain(request.args.get('domain', request.url_root))
```
Comment (glenrobson) [inline iiify/app.py]: Hardcode this to iiif.archivelab.org

**Comment by glenrobson** [comment]
Make v2 use cantaloupe.

File: iiify/resolver.py [hunk 3]
```diff
@@ -394,12 +396,12 @@ def create_manifest3(identifier, domain=None, page=None):
         for file in [f for f in originals if f['format'] in ['VBR MP3', '32Kbps MP3', '56Kbps MP3', '64Kbps MP3', '96Kbps MP3', '128Kbps MP3', 'MPEG-4 Audio', 'Flac', 'AIFF', 'Apple Lossless Audio', 'Ogg Vorbis', 'WAVE', '24bit Flac', 'Shorten']]:
             normalised_id = file['name'].rsplit(".", 1)[0]
             slugged_id = normalised_id.replace(" ", "-")
-            c_id = f"https://iiif.archivelab.org/iiif/{identifier}/{slugged_id}/canvas"
+            c_id = f"https://iiif.archive.org/iiif/{identifier}/{slugged_id}/canvas"
```
Comment (glenrobson) [inline iiify/resolver.py]: use URI prefix.

**Comment by glenrobson** [comment]
@digitaldogsbody I was having a look a the v2 manifest and although we don't directly link to cantaloupe the manifest points to the local server iiiify server e.g:

```
"service": {
    "@context": http://iiif.io/api/image/2/context.json,
    "@id": "https://archivelabs-iiif-archivelab-org-presentation-v3.ux-fnf-misc.archive.org/iiif/mms_metcalf_box_002$0",
     "profile": https://iiif.io/api/image/2/profiles/level2.json
}
```

Which calls your code that redirects to cantaloupe so I don't thing there is anything to do here.

**Comment by digitaldogsbody** [comment]
Ok good stuff! We can revisit in the future to update the v2 code to get the cantaloupe address directly rather than having to make the client make an additional request just to get the redirect

**Comment by digitaldogsbody** [comment]
Unless of course we want to obscure the canteloupe address in the manifests and serve a redirect also for v3, which would allow the address of the canteloupe deployment to move while keeping the `service` URI in the manifests static (and with a nice iiif.archive.org URL as well). I vaguely recall discussing this but I don't remember what we decided in the end.

**Comment by glenrobson** [comment]
I think it's nice to leave the iiif.archive.labs url there for now and update it if we move to iiif.archive.org/iiif/image.

**Comment by digitaldogsbody** [comment]
Agreed.

Presumably when the production service goes live with the DNS, the current Canteloupe instance (or whatever production version replaces it) will be reverse-proxied behind iiif.archive.org/iiif/image as well, so we should make a reminder to update the URL served in v3 manifests away from the current https://services-ia-iiif-cantaloupe-experiment.dev.archive.org/iiif/ that is returned.

**Pull request merged**